ods_ci/tasks/Resources/RHODS_OLM/install/oc_install.robot

```diff
@@ -117,6 +122,38 @@ Verify RHODS Installation
     ...                   label_selector=app.kubernetes.io/name=data-science-pipelines-operator
     ...                   timeout=400
   END
+  ${kserve} =    Is Component Enabled    kserve    ${DSC_NAME}
```

Comment:
The `Verify RHODS Installation` keyword is very long.  Could this code be in a new keyword, like `Install Kserve Dependencies` or similar?